src/adapter/http/src/http_server_dataset_router.rs

```diff
@@ -141,7 +144,10 @@ pub async fn platform_login_handler(
 #[utoipa::path(
     get,
     path = "/platform/token/validate",
-    responses((status = OK, body = ())),
+    responses(
+        (status = OK, body = ()),
+        (status = UNAUTHORIZED, body = ())
+    ),
     tag = "kamu",
     security(
         (),
```

Comment:
Double check here: `security(()...` means we allow anonymous access. Can there be an anonymous access in token validation call?

Reply:
Eagle eyes!

Of course, no one anonymous access is possible here -- although how we look at it. With anonymous access (without a token), we just get `UNAUTHORIZED`

Reply:
Done